adam-apis/src/main/scala/org/bdgenomics/adam/apis/java/JavaADAMContext.scala

```diff
@@ -84,6 +84,16 @@ class JavaADAMContext(val ac: ADAMContext) extends Serializable {
   }
 
   /**
+   * Loads in a coverage file. This method can load BED, NARROWPEAK, and ADAM files.
```

Comment:
NARROWPEAK &rarr; NarrowPeak as used elsewhere

What about the other feature formats (i.e. GFF3, GTF/GFF2, and IntervalList) supported by `loadFeatures`?